src/components/Button/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { View } from 'react-native'
+import { styles } from './style'
+
+const Button = ({ children }) => {
+    return (
+        <View style={styles.buttonContainer}>
+            <View style={styles.button}>
+                {children}
+            </View>
+        </View>
```

Comment:
@brandonfajardo button does not require nested Views.

outer view should use a [TouchableHighlight](https://facebook.github.io/react-native/docs/touchablehighlight.html)